wpeverest/user-registration
# Enhancement - Setting page header UI

### All Submissions:






### Changes proposed in this Pull Request:



This PR enhanced the setting page header UI which helps to organize the menu with the clean UI.

**Existing Design**

![image](https://user-images.githubusercontent.com/31344435/100958796-19fda180-3545-11eb-952e-41ad8fe0ad2d.png)

**Proposed Design**

![image](https://user-images.githubusercontent.com/31344435/100958972-65b04b00-3545-11eb-8d09-75e8fc14c1d1.png)



### How to test the changes in this Pull Request:

1. Checkout to the branch.
2. Check the setting page design.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Setting page header UI
